src/main/java/org/generationcp/middleware/service/api/BrapiView.java

```diff
@@ -5,15 +5,15 @@
  */
 public class BrapiView {
 
-	public static class BrapiV1_3 {
+	public interface BrapiV1_3 extends BrapiV1_2 {
 
 	}
 
-	public static class BrapiV1_2 {
+	public interface BrapiV1_2 {
 
 	}
 
```

Comment:
Hi @gelli-leafnode , minor comment. Move this interface above the BrapiV1_3 interface for readability purposes.